src/theme.js

```diff
@@ -53,21 +54,19 @@ function getColors(palette) {
 
 const theme = {
 	fonts: {
-		base: 'Helvetica Neue, Helvetica, Arial, sans-serif',
+		body: 'Helvetica Neue, Helvetica, Arial, sans-serif',
```

Comment:
Any particular reason to prefer `body`? I was always using `base`, because it can work for other values too — base font size, base font weight, base something else…

Reply:
Mainly for compatibility with theme-ui but I grew on it and I think we shouldn’t have used `base` since it’s too implicit. It doesn’t answer the question “what is it for”.

Reply:
We should add this to the workshop as well...

Reply:
I'm fine with the change.

To add explanation why to prefer `body` or about `theme-ui`?

Reply:
An explanation about the `body` over `base`.